Freshli.Web/wwwroot/js/site.js

```diff
@@ -0,0 +1,4 @@
+﻿// Please see documentation at https://docs.microsoft.com/aspnet/core/client-side/bundling-and-minification
+// for details on configuring this project to bundle and minify static web assets.
+
+// Write your JavaScript code.
```

Comment:
Is this file required? Can it be deleted? If not can the comments be updated to include verbiage about why it's included?

Reply:
This was generated by the `dotnet new mvc` command which was used to initially create the `Freshli.Web` project.

Reply:
I'd like for us to switch to using npm or yarn for managing dependencies and using something from the node ecosystem to bundle our code for delivery. That would replace all of the JS that was autogenerated. I'll create an issue to take care of that.

Reply:
Created issue #55.